pkg/parse/root.go

```diff
@@ -139,8 +139,6 @@ func (p *root) parseSource(ctx context.Context, state sourceState) ([]ast.FileOb
 		objs, err = validate.Hierarchical(objs, options)
 	}
 
-	metrics.RecordReconcilerErrors(ctx, "parsing", status.NonBlockingErrors(err))
```

Comment:
Instead of removing the reconciler_errors caused by parsing errors, should we replace the component with `source`?

Reply:
The parsing errors is a part of the source errors. This metric record is duplicate and does not align with the R*Sync status APIs.

Reply:
ACK